src/Models/Classes/ViewerConfigUtility.ts

```diff
@@ -1007,6 +1007,10 @@ abstract class ViewerConfigUtility {
         return color;
     }
 
+    static getValueIsWithinRange(values: unknown[], value: number): boolean {
```

Comment:
do we have teh type you defined for the values array? I thought we aliased the number | string | boolean to a ValuesType or something.

Reply:
I cast it in each function it was being used when I knew what its type was going to be. Do you suggest to change it from unknown to number | string | boolean in the schema?